ansible/vars/topo_t0-80.yml

```diff
@@ -0,0 +1,234 @@
+topology:
+  host_interfaces:
+   - 0
+   - 1
+   - 2
+   - 3
+   - 4
+   - 5
+   - 6
+   - 7
+   - 8
+   - 9
+   - 10
+   - 11
+   - 12
+   - 13
+   - 14
+   - 15
+   - 20
+   - 21
+   - 22
+   - 23
+   - 24
+   - 25
+   - 26
+   - 27
+   - 28
+   - 29
+   - 30
+   - 31
+   - 32
+   - 33
+   - 34
+   - 35
+   - 44
+   - 45
+   - 46
+   - 47
+   - 48
+   - 49
+   - 50
+   - 51
+   - 52
+   - 53
+   - 54
+   - 55
+   - 56
+   - 57
+   - 58
+   - 59
+   - 64
+   - 65
+   - 66
+   - 67
+   - 68
+   - 69
+   - 70
+   - 71
+   - 72
+   - 73
+   - 74
+   - 75
+   - 76
+   - 77
+   - 78
+   - 79
+
+  disabled_host_interfaces:
+   - 32
+   - 33
+   - 34
+   - 35
+   - 48
+   - 49
+   - 50
+   - 51
+  VMs:
+    ARISTA01T1:
+      vlans:
+        - 36
+        - 37
+      vm_offset: 0
+    ARISTA02T1:
+      vlans:
+        - 38
+        - 39
+      vm_offset: 1
+    ARISTA03T1:
+      vlans:
+        - 40
+        - 41
+      vm_offset: 2
+    ARISTA04T1:
+      vlans:
+        - 42
+        - 43
+      vm_offset: 3
```

Comment:
I would add VMs so that the 16-19 and 60-63 uplinks are also in use.

Reply:
sure Danny, Will include other uplink ports also.

Reply:
@chitra-raghavan did this PR fall out of your rader?

Reply:
@yxieca , working on this PR.

Reply:
Would you please also include a documentation of updated topology ( https://github.com/Azure/sonic-mgmt/blob/master/docs/testbed/README.testbed.Topology.md )

Reply:
Updated the README file

Reply:
thanks